.github/workflows/issue-routing-helper.yml

```
@@ -0,0 +1,37 @@
+name: Issue Routing Helper
+on:
+  issues:
+    types: [labeled]
+env:
+  # Use GH_RELEASE_PAT as github-actions bot is not allowed to ping teams
+  GH_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
+  GH_REPO: ${{ github.repository }}
+jobs:
+  route:
+    runs-on: ubuntu-latest
+    if: "startsWith(github.event.label.name, 'Team: ')"
+    steps:
+      - name: "Ensure a single 'Team: *' label with 'Status: Untriaged'"
+        run: |
+          labels_to_remove=$(gh api --paginate "/repos/$GH_REPO/labels" -q '[.[].name | select((startswith("Team: ") or startswith("Status: ")) and . != "${{ github.event.label.name }}" and . != "Status: Untriaged")] | join(",")')
+          gh issue edit ${{ github.event.issue.number }} --remove-label "$labels_to_remove" --add-label '${{ github.event.label.name }},Status: Untriaged'
+      - name: "Mention/ping assigned team for triage"
+        run: |
+          # Get team label mention name:
+          mention_target=$(gh api "/repos/$GH_REPO/labels/${{ github.event.label.name }}" -q '.description')
+          mention_slug=$(gh api "/orgs/getsentry/teams/$mention_target" -q .slug || true)
+          if [[ -z "$mention_slug" ]]; then
+            echo "Couldn't find team mention from slug, trying the label name"
+            team_label='${{ github.event.label.name }}'
+            team_name="${team_label:6}" # Strip the first 6 chars, which is the 'Team: ' part
+            team_name="${team_name// /-}" # Replace spaces with hyphens for url/slug friendliness
```

Keep the variable naming consistent with the first attempt just above?

```suggestion
            mention_target='${{ github.event.label.name }}'
            mention_target="${mention_target:6}" # Strip the first 6 chars, which is the 'Team: ' part
            mention_target="${mention_target// /-}" # Replace spaces with hyphens for url/slug friendliness
```

The `team_name` variable name in particular threw me for a loop because the API actually takes the slug not the name.

---

I'll rename it to `team_slug` then. I don't like reusing `mention_target` as that doesn't convey what I'm trying to do here.